src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+const list = document.querySelectorAll('li');
```

Comment:
If you get all 'li' elements it's listItems, not list
```suggestion
const listItems = document.querySelectorAll('li');
```